msgraph/models.go

```diff
@@ -540,6 +540,15 @@ type BaseNamedLocation struct {
 	ModifiedDateTime *time.Time  `json:"modifiedDateTime,omitempty"`
 }
 
+type ClaimsMappingPolicy struct {
+	DirectoryObject
+	Definition            *[]string `json:"definition,omitempty"`
+	Description           *string   `json:"description,omitempty"`
+	DisplayName           *string   `json:"displayName,omitempty"`
+	ID                    *string   `json:"id,omitempty"`
```

Comment:
Can we remove the `ID` field from the `ClaimsMappingPolicy` type as this will enable the `ID` field to be embedded from `DirectoryObject`

Reply:
Thank you for the feedback. This has been adjusted @manicminer.